app/lib/activitypub/activity/create.rb

```diff
@@ -54,6 +51,7 @@ def status_params
       visibility: visibility_from_audience,
       thread: replied_to_status,
       conversation: conversation_from_uri(@object['conversation']),
+      media_attachments: process_attachments,
```

Comment:
In this case, it is not good to store the media in the transaction.

Reply:
Save the return value of `process_status_params` to the `status_params` variable beforehand and pass it as an argument of `Status.create`?

Reply:
It looks good!